### PR TITLE
Feat: Added Modal to CL

### PR DESCRIPTION
**Pull request by zoef-yt**
Added two types of modal to CL. 
1. Simple modal.
2. Modal with input.

### CODE REVIEW

File: pages/components/modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
+.modal-opened {
+	/* display: none; */
```
Comment (Shubhamkr2610) [inline pages/components/modal/modal.css]: remove the unused code

File: pages/components/modal/modal.html [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Modal</title>
+		<link rel="stylesheet" href="../../../styles.css" />
+	</head>
+	<body class="showcase-body">
+		<h1 class="showcase-header">Modal</h1>
+		<hr />
```
Comment (Shubhamkr2610) [inline pages/components/modal/modal.html:12]: I  think  there is typo error. Do correction if I am not wrong

File: pages/components/modal/modal.css [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+	position: fixed;
+	top: 0;
+	left: 0;
+	width: 100%;
+	height: 100%;
+	background-color: rgba(0, 0, 0, 0.5);
```
Comment (DivyanshSareen) [inline pages/components/modal/modal.css]: can use a variable for background-color

**Review by DivyanshSareen — APPROVED** [review]

**Review by Shubhamkr2610 — APPROVED** [review]
Looks Good to me. Ready to merge

**Review by Omkar-Ghate — APPROVED** [review]
Great work. Good to merge.

**Pull request merged**